integrations/tensorflow/compiler/dialect/tf_strings/ir/types.h

```diff
@@ -41,10 +41,7 @@ class TFStringsType : public Type {
  public:
   using Type::Type;
 
-  static bool classof(Type type) {
-    return type.getKind() >= TFStringsTypes::FIRST_USED_STRINGS_TYPE &&
-           type.getKind() <= TFStringsTypes::LAST_USED_STRINGS_TYPE;
-  }
+  static bool classof(Type type);
```

Comment:
This doesn't need to be implemented somewhere?

Reply:
I put it in tf_strings/ir/dialect.cpp.

Reply:
Whoops not sure how I missed that.